src/pages/About/KyberSwapGeneralIntro.tsx

```diff
@@ -9,7 +9,7 @@ import { MoneyBagOutline } from 'components/Icons'
 import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
 import useTheme from 'hooks/useTheme'
 
-const KyberSwapGeneralIntro = () => {
+const KyberSwapGeneralIntro = ({ isSolana }: { isSolana: boolean }) => {
```

Comment:
I think it will be better to not pass it as props, but make it call useActiveWeb3 directly

Reply:
to reduce dependancy between it and its parent

Reply:
in many cases, pass prop is not bad if only pass 1 layer, more than 1 will become prop drilling. Use hook in other hand make code look duplicated, more function calls. 
In this case, KyberSwapGeneralIntro  only a simple render component, its ok to take a prop from parent. Use hook mean should run isSolana function one more time.